README.rst

```diff
@@ -495,6 +499,13 @@ Choose from the list of available fixers:
                         An empty line feed should
                         precede a return statement.
 
+* **scalar_cast** [symfony]
```

Comment:
> So, as both form could be considered as correct

That's what I have a problem with though, lol. IMO, there can only be one actual name.

I agree we should probably just choose `int` and `bool` since those seem to be conventionally used.